storybook/manager-head.html

```diff
@@ -15,8 +15,22 @@
   #storybook-explorer-searchfield {
     background-color: white;
   }
+</style>
+
+<script>
+  const storybookDocPath = "kaio-staging"
+  const production = new URLPattern("http{s}?://cultureamp.design");
+  currentUrl = window.location.href;
 
-  data-title["Legacy"] {
-    background-color: red;
+  const shouldHideKAIOStories = production.test(currentUrl);
```

Comment:
Shouldn't we just check the env variable? 🤔

Reply:
Hm, I didn't think I could access that here

Reply:
https://storybook.js.org/docs/react/configure/environment-variables :)

Reply:
Hm, it's undefined here. My guess is that Storybook doesn't actually bundle this code, it just uses it as a template.